context/app/static/js/pages/Publication/Publication.jsx

```diff
@@ -61,7 +62,8 @@ function Publication({ publication, vignette_json }) {
       {shouldDisplaySection.visualizations && (
         <PublicationsVisualizationSection vignette_json={vignette_json} uuid={uuid} />
       )}
-      <Files files={files} uuid={uuid} hubmap_id={hubmap_id} />
+      {shouldDisplaySection.files && <Files files={files} uuid={uuid} hubmap_id={hubmap_id} />}
+      <BulkDataTransfer files={files} uuid={uuid} hubmap_id={hubmap_id} />
```

Comment:
Same as above, just for consistency 👍 

```suggestion
      {shouldDisplaySection.bulkDataTransfer && <BulkDataTransfer files={files} uuid={uuid} hubmap_id={hubmap_id} />}
```